ISLE/isleapp.cpp

```diff
@@ -540,6 +545,45 @@ void IsleApp::LoadConfig()
 
 	dictionary* dict = iniparser_load(iniConfig);
 
+	// [library:config]
+	// Load sane defaults if dictionary failed to load
+	if (!dict) {
+		SDL_LogInfo(SDL_LOG_CATEGORY_APPLICATION, "Loading sane defaults");
+		FILE* iniFP = fopen(iniConfig, "w");
+
+		if (!iniFP) {
+			SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "Failed to write config at '%s': %s", iniConfig, strerror(errno));
+			fclose(iniFP);
```

Comment:
Avoid a null pointer dereference.
```suggestion
			return;
```

Reply:
Would this still load the HD and CD paths?

Reply:
I might go back to my previous method then.

Which would be creating a blank dictionary with `dictionary_create(0)`

then write to a file afterwards. with `iniparser_dump_ini`